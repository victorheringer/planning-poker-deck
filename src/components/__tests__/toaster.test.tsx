import { render } from "@testing-library/react";
import Toaster from "../toaster";

describe("Toaster", () => {
  it("should not render anything if show is false", () => {
    const tree = render(<Toaster show={false} text="test" />).asFragment();
    expect(tree).toMatchSnapshot();
  });

  it("should render correctly if show is false", () => {
    const tree = render(<Toaster show text="test toaster" />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
