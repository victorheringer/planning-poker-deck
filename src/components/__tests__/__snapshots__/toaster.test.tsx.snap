// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toaster should not render anything if show is false 1`] = `<DocumentFragment />`;

exports[`Toaster should render correctly if show is false 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL ddBRey"
  >
    <p
      class="sc-gsDKAQ gcDiLF"
    >
      test toaster
    </p>
  </div>
</DocumentFragment>
`;
